app/script/cryptography/CryptographyRepository.js

```diff
@@ -50,15 +50,49 @@ z.cryptography.CryptographyRepository = class CryptographyRepository {
   }
 
   /**
+   * Initializes the repository by loading an existing cryptobox.
+   * @param {Object} database - Database object
+   * @returns {Promise} Resolves after initialization
+   */
+  create_cryptobox(database) {
+    return this._init(database)
+      .then(() => this.cryptobox.create());
+  }
+
+  /**
+   * Initializes the repository by creating a new cryptobox.
```

Comment:
`cryptobox` should be capitalized: https://github.com/wireapp/cryptobox#cryptobox